tools/barista/src/builder/components.ts

```diff
@@ -73,22 +73,39 @@ function setMetadataDefaults(baristaMetadata: any): BaSinglePageMeta {
   return metadataWithDefaults;
 }
 
+function getPaths(path: string, fileList: string[]): string[] {
+  let files = readdirSync(path);
+
+  files.forEach(function(file: string): void {
+    const filePath = join(path, file);
+    if (lstatSync(filePath).isDirectory()) {
+      // Only grab those dirs that include a README.md and a barista.json
+      if (
+        existsSync(join(filePath, 'README.md')) &&
+        existsSync(join(filePath, 'barista.json'))
+      ) {
+        fileList.push(filePath);
+      }
+      getPaths(filePath, fileList);
```

Comment:
i guess the paths returned here should also be pushed

Reply:
this should do the trick. 
```suggestion
      fileList.push(...getPaths(filePath, fileList));
```